packages/core/upload/admin/src/components/FolderList/FolderList.js

```diff
@@ -73,7 +78,7 @@ export const FolderList = ({
                 size={size}
               >
                 <FolderCardBody>
-                  <FolderCardLink to={url}>
+                  <FolderCardBodyAction to={url}>
```

Comment:
Does nesting `FolderCardBodyAction` into `FolderCardBody` mean the folder icon is not linked? I'd prefer if the whole visible area of the `FolderCard` could be clickable. Wdyt?

Reply:
the icon is clickable, as the whole `FolderCard`, thanks to the `onClick` we pass to `FolderCard`